cmd/vpp-agent-ctl/topology/static_routing.sh

```diff
@@ -16,7 +16,7 @@ RPD_NAME="rpd"
 #
 
 # VSWITCH - add static route to 112.1.1.3/32 via 8.42.0.1
-vpp-agent-ctl -put /vnf-agent/${VSWITCH_NAME}/vpp/config/v1/vrf/0/fib/112.1.1.3/32/8.42.0.1 - << EOF
+../vpp-agent-ctl -put /vnf-agent/${VSWITCH_NAME}/vpp/config/v1/vrf/0/fib/112.1.1.3/32/8.42.0.1 - << EOF
```

Comment:
This will require to always run this from this folder. Compared to previous case which used installed version of vpp-agent-ctl and the script could be run from anywhere.

Reply:
It was 'refactored' when I moved it to different folder

Reply:
Not sure I follow.